lib/build_script.rb

```diff
@@ -25,14 +25,13 @@ def run!(cmd)
 
 # Build logic in a method
 def run_build_script(
-  io: $stdout,
+  architecture:, io: $stdout,
```

Comment:
```suggestion
  architecture:, 
  io: $stdout,
```

Reply:
Weird that standardrb didn't have an opinion about that